systemd/init

```diff
@@ -0,0 +1,51 @@
+#!/bin/busybox sh
+
+set -e
+
+# Init for Wolfi OS
+# - Environment variables:
+#     VERBOSE=1 for debugging
+#     CMDLINE=/path/to/file to override /proc/cmdline, useful for testing
+# - Mount /proc, /sys, /dev if not already mounted
+# - Allow for override of path to /init with init=/path/to/init on kernel /proc/cmdline
+# - Allow for uuid or device path for root filesystem with root= on kernel /proc/cmdline
+# - Mount root, if specified and found, switch_root to it, and run systemd as init
+# - Otherwise, try to run systemd as init in the initrd
+
+# Verbose debugging
+info() { [ "$VERBOSE" = "1" ] && printf "INFO: %s\n" "$*" 1>&2; }
+
+# Override /proc/cmdline path as environment variable, useful for testing
+[ -z "$CMDLINE" ] && CMDLINE="/proc/cmdline"
+info "cmdline = [$(cat $CMDLINE)]"
+
+# Setup our system mounts if needed
+mkdir -p /proc /sys /dev
+findmnt /proc >/dev/null 2>&1 || mount -t proc none /proc
```

Comment:
we could add `nodev,nosuid,hidepid=2` to harden the mountpoint

Reply:
Sure, I'm good with that.